Program.cs

```diff
@@ -1,10 +1,18 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿// Convert decimal to hexadecimal and binary
+
+// To Hexadecimal
+
+Console.Write ("Enter a decimal value: ");
+int input = int.Parse (Console.ReadLine ());
+string hexvalue = input.ToString ("X");
+Console.WriteLine (hexvalue);
+
+// To binary 
+
+string binarynumber = "";
+while (input > 0) {
+   int remainder = input % 2;
+   binarynumber = Convert.ToString (remainder) + binarynumber;

```

Comment:
No need to use Convert.tostring you can add integer to a string which will append the integer to the string. No need of a remainder variable if you are using one instance you can pass the expression itself.